src/ios/CDVStatusBar.m

```diff
@@ -195,6 +195,14 @@ - (void) _ready:(CDVInvokedUrlCommand*)command
     }
 }
 
+- (void) height:(CDVInvokedUrlCommand*)command
+{
+    double statusBarHeight = [UIApplication sharedApplication].statusBarFrame.size.height;
+    CDVPluginResult* result = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDouble:statusBarHeight];
+    [result setKeepCallbackAsBool:YES];
```

Comment:
Someone who has a bit more knowledgeable about iOS native development than me might be able shed more like but I think this `setKeepCallbackAsBool` is intended to be used to keep a reference of the callback so it can be invoked several times? Is this necessary here?

Reply:
Yes that is the intend, otherwise the callback will be deleted on the js side.
https://github.com/apache/cordova-js/blob/0608bf932289c143405e90dc8edd3449f78f5fdc/src/cordova.js#L230-L233

Reply:
Thanks for the feedback.

@breautek, if you want to merge this, I've created a rebased version of this branch, you can merge that one if you want, (with attribution to the original authors)

> https://github.com/SchnWalter/cordova-plugin-statusbar/tree/134-rebased/expose-StatusBar-height